linux_os/guide/system/logging/ensure_rsyslog_log_file_configuration/rsyslog_files_permissions/rule.yml

```diff
@@ -30,6 +30,7 @@ references:
     cis@debian8: 5.1.4
     anssi@debian8: NT28(R36)
     anssi@debian9: NT28(R36)
+    anssi@debian10: NT28(R36)
```

Comment:
Since all the anssi mappings are the same, please collapse into a single entry e.g. `anssi: NT28(R36)`